src/include/daos_errno.h

```diff
@@ -43,143 +43,209 @@ extern "C" {
 	/** PMIx layer error */						\
-	ACTION(DER_PMIX,		(DER_ERR_GURT_BASE + 23))	\
+	ACTION(DER_PMIX,		(DER_ERR_GURT_BASE + 23),	\
+	       PMIx layer error)					\
 	/** IV callback - cannot handle locally */			\
-	ACTION(DER_IVCB_FORWARD,	(DER_ERR_GURT_BASE + 24))	\
+	ACTION(DER_IVCB_FORWARD,	(DER_ERR_GURT_BASE + 24),	\
+	       IV callback - cannot handle locally)			\
 	/** miscellaneous error */					\
-	ACTION(DER_MISC,		(DER_ERR_GURT_BASE + 25))	\
+	ACTION(DER_MISC,		(DER_ERR_GURT_BASE + 25),	\
+	       Miscellaneous error)					\
 	/** Bad path name */						\
-	ACTION(DER_BADPATH,		(DER_ERR_GURT_BASE + 26))	\
+	ACTION(DER_BADPATH,		(DER_ERR_GURT_BASE + 26),	\
+	       Bad path name)						\
 	/** Not a directory */						\
-	ACTION(DER_NOTDIR,		(DER_ERR_GURT_BASE + 27))	\
+	ACTION(DER_NOTDIR,		(DER_ERR_GURT_BASE + 27),	\
+	       Not a directory)						\
 	/** corpc failed */						\
-	ACTION(DER_CORPC_INCOMPLETE,	(DER_ERR_GURT_BASE + 28))	\
+	ACTION(DER_CORPC_INCOMPLETE,	(DER_ERR_GURT_BASE + 28),	\
+	       Corpc failed)						\
 	/** no rank is subscribed to RAS */				\
-	ACTION(DER_NO_RAS_RANK,		(DER_ERR_GURT_BASE + 29))	\
+	ACTION(DER_NO_RAS_RANK,		(DER_ERR_GURT_BASE + 29),	\
+	       No rank is subscribed to RAS)				\
 	/** service group not attached */				\
-	ACTION(DER_NOTATTACH,		(DER_ERR_GURT_BASE + 30))	\
+	ACTION(DER_NOTATTACH,		(DER_ERR_GURT_BASE + 30),	\
+	       Service group not attached)				\
 	/** version mismatch */						\
-	ACTION(DER_MISMATCH,		(DER_ERR_GURT_BASE + 31))	\
+	ACTION(DER_MISMATCH,		(DER_ERR_GURT_BASE + 31),	\
+	       Version mismatch)					\
 	/** rank has been evicted */					\
-	ACTION(DER_EVICTED,		(DER_ERR_GURT_BASE + 32))	\
+	ACTION(DER_EVICTED,		(DER_ERR_GURT_BASE + 32),	\
+	       Rank has been evicted)					\
 	/** user-provided RPC handler didn't send reply back */		\
-	ACTION(DER_NOREPLY,		(DER_ERR_GURT_BASE + 33))	\
+	ACTION(DER_NOREPLY,		(DER_ERR_GURT_BASE + 33),	\
+	       User provided RPC handler did not send reply back)	\
 	/** denial-of-service */					\
-	ACTION(DER_DOS,			(DER_ERR_GURT_BASE + 34))       \
+	ACTION(DER_DOS,			(DER_ERR_GURT_BASE + 34),       \
+	       Denial of service)					\
 	/** Incorrect target for the RPC  */				\
-	ACTION(DER_BAD_TARGET,		(DER_ERR_GURT_BASE + 35))	\
+	ACTION(DER_BAD_TARGET,		(DER_ERR_GURT_BASE + 35),	\
+	       Incorrect target for the RPC)				\
 	/** Group versioning mismatch */				\
-	ACTION(DER_GRPVER,		(DER_ERR_GURT_BASE + 36))
+	ACTION(DER_GRPVER,		(DER_ERR_GURT_BASE + 36),	\
+	       Group versioning mismatch)
 	/** TODO: add more error numbers */
 
 #define D_FOREACH_DAOS_ERR(ACTION)					\
 	/** Generic I/O error */					\
-	ACTION(DER_IO,			(DER_ERR_DAOS_BASE + 1))	\
+	ACTION(DER_IO,			(DER_ERR_DAOS_BASE + 1),	\
+	       Input / output error)					\
 	/** Memory free error */					\
-	ACTION(DER_FREE_MEM,		(DER_ERR_DAOS_BASE + 2))	\
+	ACTION(DER_FREE_MEM,		(DER_ERR_DAOS_BASE + 2),	\
+	       Memory free error)					\
```

Comment:
Not sure what this error is...it's not used anywhere but the unit test.   Regardless, we shouldn't remove such things so we don't break anything.